slave-base/contrib/bin/run-jnlp-client

```diff
@@ -8,7 +8,32 @@
 # The directory has to be writeable for the user that the container is running
 # under.
 
-#NOTE:  periodically check https://ce-gitlab.usersys.redhat.com/ce/jboss-dockerfiles/blob/develop/scripts/os-java-run/added/java-default-options for updates
+# NOTE:  periodically check https://ce-gitlab.usersys.redhat.com/ce/jboss-dockerfiles/blob/develop/scripts/os-java-run/added/java-default-options for updates
+
+validateConnection() {
+  i=1
+  echo "INFO: $(date +%T)[$i]: Checking connection to $JENKINS_HOSTNAME"
+  until curl -L --max-time $INTERVAL --connect-timeout $INTERVAL "$JENKINS_HOSTNAME"; do
```

Comment:
JENKINS_URL instead of hostname

Reply:
JENKINS_URL is a variable exported/used by kubernetes client. JENKINS_HOSTNAME is better.